sphinx/environment/__init__.py

```diff
@@ -407,25 +396,22 @@ def relfn2path(self, filename, docname=None):
             enc_rel_fn = rel_fn.encode(sys.getfilesystemencoding())
             return rel_fn, path.abspath(path.join(self.srcdir, enc_rel_fn))
 
+    @property
+    def found_docs(self):
+        # type: () -> Set[unicode]
+        """contains all existing docnames."""
```

Comment:
Note: `env.found_docs` has used for long time. So I'd not like to deprecate this. It might be a big breaking change!

Reply:
OK. It would be better if the reason was written in the docstring of this function.

Reply:
It means your 'Note'

> Note: `env.found_docs` has used for long time. So I'd not like to deprecate this. It might be a big breaking change!

Reply:
I feel strange. Are you saying to add "this is not deprecated yet"?

Reply:
Since you wrote a note for developers on PR, I thought it was worth keeping in the comments of the code. I think that the part of "(Removing this method) might be a big breaking change" is an important note.

Reply:
This is a public API which has been longly used.
http://www.sphinx-doc.org/en/master/extdev/envapi.html#sphinx.environment.BuildEnvironment.found_docs

So removing this will bring big breaking change as you know.
I don't think it is not important decision. To keep API stable is our responsibility. I believe Not to remove public APIs is not special things for us. Is it really need to add comment? Should I add to all promised APIs to the comment?

Reply:
My concern is what will be given for the readers of our code or document. Removing public APIs  will always bring breaking change. So I think the comment is meaningless.

Reply:
I think this discussion is stalled. But it's trivial. So I'm merging this now.
If you still have opinion, please let me know in this thread. I'll respond to your comment even after merged.

Thanks,